src/gui_common/tooltip/ToolTipManager.cs

```diff
@@ -516,4 +542,98 @@ private void HideAllToolTips()
             }
         }
     }
+
+    private void CreateOrganelleTooltips()
+    {
+        var packedTooltip = GD.Load<PackedScene>("res://src/microbe_stage/editor/tooltips/SelectionMenuToolTip.tscn");
+
+        foreach (var organelle in SimulationParameters.Instance.GetAllOrganelles())
+        {
+            var tooltip = packedTooltip.Instantiate<SelectionMenuToolTip>();
+
+            Instance.AddToolTip(tooltip, "organelleSelection");
+
+            tooltip.Description = organelle.Description;
+            tooltip.MutationPointCost = organelle.MPCost;
+            tooltip.Name = organelle.InternalName;
+            tooltip.DisplayName = organelle.Name;
+            tooltip.RequiresNucleus = organelle.RequiresNucleus;
+            tooltip.ThriveopediaPageName = organelle.InternalName;
+            tooltip.ProcessesDescription = organelle.ProcessesDescription == null ?
+                string.Empty :
+                organelle.ProcessesDescription;
+
+            FillOrganelleToolTipModifierInfo(organelle, tooltip);
+        }
+    }
+
+    private void FillOrganelleToolTipModifierInfo(OrganelleDefinition organelle, ICustomToolTip tooltip)
+    {
+        if (tooltip is not SelectionMenuToolTip selectionMenuTooltip)
+            return;
+
+        if (organelle.Components.Storage != null)
+        {
+            // Modifier info is filled info later
+            selectionMenuTooltip.AddModifierInfo(string.Empty, string.Empty, 0,
+                "res://assets/textures/gui/bevel/StorageIcon.png", "storage");
+        }
+
+        if (organelle.Components.Lysosome != null)
+        {
+            selectionMenuTooltip.AddModifierInfo(string.Empty, string.Empty, 0,
+                "res://assets/textures/gui/bevel/DigestionSpeedIcon.png", "digestionSpeed");
+
+            selectionMenuTooltip.AddModifierInfo(string.Empty, string.Empty, 0,
+                "res://assets/textures/gui/bevel/DigestionIcon.png", "digestionEfficiency");
+        }
+
+        if (organelle.Components.Movement != null)
+        {
+            selectionMenuTooltip.AddModifierInfo(string.Empty, string.Empty, 0,
+                "res://assets/textures/gui/bevel/SpeedIcon.png", "speed");
+        }
+
+        UpdateModifierInfoWithTranslations(organelle, selectionMenuTooltip);
+    }
+
+    private void UpdateModifierInfoWithTranslations(OrganelleDefinition organelle, ICustomToolTip tooltip)
+    {
+        if (tooltip is not SelectionMenuToolTip selectionMenuTooltip)
+            return;
+
+        var modifierInfo = selectionMenuTooltip.GetModifierInfo("storage");
+
+        if (modifierInfo == null)
+            return;
```

Comment:
Doesn't this cause this to stop working if an earlier stat is missing (due to this return)?

I'll fix myself all the problems I see with this PR to get this done as reviewing this is starting to take more time than it would have taken me to do this from scratch.